web/src/modules/centre/pages/CentreDashboardSummaryReconciliationPage.vue

```diff
@@ -46,6 +47,23 @@
             </v-tooltip>
             <span v-else>{{ formatMoney(centsToDollars(expense.amountInCents)) }}</span>
           </td>
+          <td class="text-right">
+            <v-tooltip
+              v-if="employee.includesEstimates"
+              bottom
+            >
+              <template #activator="{ props }">
+                <span v-bind="props">
+                  {{ formatMoney(centsToDollars(employee.amountInCents)) }}
+                  <sup class="asterisk-icon">
+                    <v-icon size="small">mdi-asterisk-circle-outline</v-icon>
+                  </sup>
+                </span>
+              </template>
+              <span class="text-white">This amount includes estimated values.</span>
+            </v-tooltip>
+            <span v-else>{{ formatMoney(centsToDollars(employee.amountInCents)) }}</span>
+          </td>
```

Comment:
@datajohnson Is the asterisk supposed to show up in the top right corner now?

![image](https://github.com/ytgov/elcc-data-management/assets/23045206/44e8cd15-9764-4146-8e20-41b96b8c2fe8)